### PR TITLE
[OBSDEF-4343] IAM UI: Adding password read-only in Datagrid row

### PR DESCRIPTION
**Pull request by m-manaswini**
Refer - https://jira.cec.lab.emc.com/browse/OBSDEF-4343

Adding Read-only password to datagrid row

As per required Acceptance Criteria
1. Password has been aligned in datagrid row.
2. Password is made readonly 

Refer image for - **Read only password in Datagrid**
image

Refer GIF for - **Read-only password**
OBSDEF - 4343 - Story_for_Read_only_password

### CODE REVIEW

File: src/forms/password/Password.tsx [hunk 1]
```diff
@@ -39,6 +39,8 @@ type PasswordProps = {
     unmask?: boolean; // if true renders eye icon to hide/show or mask/unmask password
     pattern?: string; // specifies a regular expression that element's value is checked against
     dataqa?: string; //quality engineering testing field
+    readOnly?: boolean; //specifies if password is readonly
+    formStyle?: any; //styling for form
```
Comment (akate1) [inline src/forms/password/Password.tsx:43]: formStyle -> style

**Review by akate1 — COMMENTED** [review]
Please increase version in package.json

File: src/forms/password/Password.tsx (hunk 1 above)
Comment (m-manaswini) [reply src/forms/password/Password.tsx:43]: @akate1 

style - is used with <div className={utils.classNames(["clr-input-group", className])} style={style}>
and
formStyle - is used with div className={utils.classNames(["clr-form-control", label && "clr-row"])} style={formStyle}> for alignment in grid

Comment (akate1) [reply src/forms/password/Password.tsx:43]: @m-manaswini we don't use style in plugin UI. We will handle this alignment issue in datagrid by overriding existing CSS. So please remove `formStyle  `prop

Comment (m-manaswini) [reply src/forms/password/Password.tsx]: Removed` formStyle` prop

File: src/datagrid/DataGridValues.tsx [hunk 2]
```diff
@@ -120,6 +121,52 @@ export const customRows = [
...
+                cellData: (
+                    <div>
+                        <Password
+                            name="Password"
+                            value="Brynn-pass"
+                            minPasswordLength={8}
+                            readOnly={true}
+                            style={{border: "none"}}
+                            formStyle={{marginTop: "0rem"}}
+                        />
```
Comment (akate1) [inline src/datagrid/DataGridValues.tsx:162]: remove `formStyle` from here as well

Comment (m-manaswini) [reply src/datagrid/DataGridValues.tsx:161]: removed

File: src/datagrid/DataGrid.stories.tsx [hunk 3]
```diff
@@ -17,6 +17,7 @@ import {
     normalColumns,
```
Comment (maheshkulkarni4) [inline src/datagrid/DataGrid.stories.tsx:17]: This PR needs rebased after Arpita's PR merge, the names of files seems changing

**Review by maheshkulkarni4 — APPROVED** [review]
LGTM.

Merge this first and then @akate1 PR

**Review by akate1 — CHANGES_REQUESTED** [review]
Need to solve some CSS issues

File: src/datagrid/DataGrid.stories.tsx (hunk 3 above)
Comment (m-manaswini) [reply src/datagrid/DataGrid.stories.tsx:17]: Rebased PR

**Review by akate1 — APPROVED** [review]
LGTM

**Comment by m-manaswini** [comment]
> Need to solve some CSS issues

CSS issues fixed

**Pull request merged**